### ISSUE TITLE
Question on FuseSupportedOps pass

### ISSUE THREAD

**Issue by vdantu** [pytorch/tvm#132]
I was just playing around with this optimization pass with the main PyTorch project. I was curious to understand why [control flow operators are not handled here](https://github.com/pytorch/tvm/blob/master/torch_tvm/fusion_pass.cpp#L30). What was the issue? I had lodged a question on the [PyTorch discuss forum](https://discuss.pytorch.org/t/question-on-customfusegraph-graph-fuser-pytorch-tvm-pytorch-glow/58627/3?u=vdantu) about the issue I was facing in my experimentation. Would anyone here be able to help with it?

TL;DR : In my experimentation, I was finding that subgraph_utils in PyTorch repo doesn't handle merging of nodes, when the inputs of the node belongs to outputs of another block's nodes. Is this understanding correct? Any pointers would be highly appreciated :)

**Comment by lly-zero-one** [comment]
@serhaty I think you saw the same issue and was working on a fix?

**Comment by vdantu** [comment]
@lly-zero-one : Thanks for responding. Is there any communication channel (slack/messenger) thats open to public? I would love to join it and try to see if my understanding is right.
